templates/patterns/page_header/page_header.ui_patterns.yml

```diff
@@ -49,8 +49,8 @@ page_header:
           label: "Announcements"
     infos:
       type: "array"
-      label: "Infos"
-      description: "A list of infos of the current page."
+      label: "Infos (deprecated)"
+      description: "A list of infos of the current page. This was deprecated from ECL 2.30."
```

Comment:
<img width="877" alt="deprecated" src="https://user-images.githubusercontent.com/22908988/91181548-bff17280-e6f1-11ea-8041-bd9b62231215.png">
In other deprecated patterns we only mentioned the major version of ecl. We could change to This was deprecated in ECL 2.x.

Reply:
Mentioning that this element deprecated from ECL 2.x meant really that such element deprecated from version 2.x (during upgrade from ECL 1 to ECL 2). In this particular case, ECL really doesn't support this element anymore only from 2.30 version.